tests/test_word_tokenize.py

```diff
@@ -7,6 +7,7 @@
 
 
 SENTENCE1 = "吾輩は猫である"
+SENTENCE2 = "医薬品安全管理責任者"
```

Comment:
This sentence is taken from the official documentation.
(https://github.com/WorksApplications/SudachiPy#as-a-python-package)